elide-contrib/elide-dynamic-config-helpers/src/main/resources/templates/table.hbs

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.dynamicconfig.model;
+
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.annotation.Exclude;
+import com.yahoo.elide.annotation.ReadPermission;
+import com.yahoo.elide.datastores.aggregation.annotation.Cardinality;
+import com.yahoo.elide.datastores.aggregation.annotation.CardinalitySize;
+import com.yahoo.elide.datastores.aggregation.annotation.DimensionFormula;
+import com.yahoo.elide.datastores.aggregation.annotation.MetricFormula;
+import com.yahoo.elide.datastores.aggregation.annotation.Join;
+import com.yahoo.elide.datastores.aggregation.annotation.Meta;
+import com.yahoo.elide.datastores.aggregation.annotation.Temporal;
+import com.yahoo.elide.datastores.aggregation.annotation.TimeGrainDefinition;
+import com.yahoo.elide.datastores.aggregation.metadata.enums.TimeGrain;
+import com.yahoo.elide.datastores.aggregation.queryengines.sql.annotation.FromSubquery;
+import com.yahoo.elide.datastores.aggregation.queryengines.sql.annotation.FromTable;
+
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import lombok.Data;
+
+import java.util.Date;
+import javax.persistence.Column;
+import javax.persistence.Id;
+
+/**
+ * A root level entity for testing AggregationDataStore.
+ */
+@Cardinality(size = CardinalitySize.{{#toUpperCase cardinality}}{{/toUpperCase}})
+@EqualsAndHashCode
+@ToString
+@Data
+{{#if table}}@FromTable(name = "{{#if schema}}{{schema}}.{{/if}}{{table}}"){{/if}}
+{{#if sql}}@FromSubquery(sql = "{{sql}}"){{/if}}
```

Comment:
Can this be part of an else statement with respect to `{{#if table}}`

Reply:
As per schema it should be one of {table, sql, extend} so both table and sql can be undefined

Reply:
changed to use if-else for table and sql